helpdesk/settings.py

```diff
@@ -231,12 +228,24 @@
 HELPDESK_USE_HTTPS_IN_EMAIL_LINK = getattr(
     settings, 'HELPDESK_USE_HTTPS_IN_EMAIL_LINK', settings.SECURE_SSL_REDIRECT)
 
-HELPDESK_TEAMS_MODEL = getattr(
-    settings, 'HELPDESK_TEAMS_MODEL', 'pinax_teams.Team')
-HELPDESK_TEAMS_MIGRATION_DEPENDENCIES = getattr(settings, 'HELPDESK_TEAMS_MIGRATION_DEPENDENCIES', [
+# Default to True for backwards compatibility
+HELPDESK_TEAMS_MODE_ENABLED = False if hasattr(settings, 'HELPDESK_TEAMS_MODE_ENABLED') and getattr(
+    settings, 'HELPDESK_TEAMS_MODE_ENABLED', True) is False else True
+if HELPDESK_TEAMS_MODE_ENABLED:
+    HELPDESK_TEAMS_MODEL = getattr(
+        settings, 'HELPDESK_TEAMS_MODEL', 'pinax_teams.Team')
+    HELPDESK_TEAMS_MIGRATION_DEPENDENCIES = getattr(settings, 'HELPDESK_TEAMS_MIGRATION_DEPENDENCIES', [
                                                 ('pinax_teams', '0004_auto_20170511_0856')])
-HELPDESK_KBITEM_TEAM_GETTER = getattr(
-    settings, 'HELPDESK_KBITEM_TEAM_GETTER', lambda kbitem: kbitem.team)
+    HELPDESK_KBITEM_TEAM_GETTER = getattr(
+        settings, 'HELPDESK_KBITEM_TEAM_GETTER', lambda kbitem: kbitem.team)
+else:
+    HELPDESK_TEAMS_MODEL = settings.AUTH_USER_MODEL
+    HELPDESK_TEAMS_MIGRATION_DEPENDENCIES = []
+    HELPDESK_KBITEM_TEAM_GETTER = lambda _: None
+
+# show knowledgebase links?
+# If Teams mode is enabled then it has to be on
+HELPDESK_KB_ENABLED = True if HELPDESK_TEAMS_MODE_ENABLED else getattr(settings, 'HELPDESK_KB_ENABLED', True)
```

Comment:
Isn't it possible to have Teams mode enabled and not wanting to have KB links enabled as well?

Reply:
I have not looked into it too deeply but the HELPDESK_KB_ENABLED setting seems to deeply disable KB functionality which teams relies on because of the way it is implemented and relies on being able to set up and mange KB items.
@timthelion - do you know unequivocally if HELPDESK_KB_ENABLED must be True for teams to work?